### DISCUSSION TITLE
[tests] Share your ideas here for tests in map2loop!

### DISCUSSION THREAD

**Discussion by lachlangrose** [Loop3D/map2loop#77] [Ideas]
This is an ongoing issue to keep track of tests that need to be added to map2loop

- [ ] intrusions: conformable stratigraphy with a dyke running through. Expected result is the contacts of stratigraphy near the dyke are ignored
- [ ] colour maps: test where clut file doesn't exist, where it is missing units, where it has invalid colour codes and where it is valid
- [ ] missing files: missing files for all inputs of m2l, either raise FileNotFound exception or continue if the file is not critical. 
- [ ] faults: make sure that the basal contacts are cropped near the faults, displacement calculation is relevant

**Comment by AngRodrigues** [comment]
- [ ] add tests for save into LPF. Should check the headers, and column dtypes as well.

**Comment by AngRodrigues** [comment]
- [x] move from having tests that use data from Loop server, as sometimes connection to GA times out and test fails

**Reply by AngRodrigues** [reply]
Fixed by skipping the test if server not available.
